server/index.js

```diff
@@ -30,6 +31,16 @@ app.get('/api/trip', (req, res, next) => {
     .catch(err => { next(err); });
 });
 
+app.get('/api/places/:id', (req, res, next) => {
+  const ApiKey = process.env.REACT_APP_GOOGLE_KEY;
```

Comment:
`REACT_APP_` is a conventional prefix when using `create-react-app` to build your project. You can just use `GOOGLE_KEY`. Is this different from your maps key? It'd be a good idea to update your `.env.example` with a dummy value so that it's understood by another developer what environment variables are required without having to search the whole codebase.